packages/jest-runtime/src/script_transformer.js

```diff
@@ -152,9 +152,10 @@ export default class ScriptTransformer {
   }
 
   _instrumentFile(filename: Path, content: string): string {
-    return babelTransform(content, {
+    const result = babelTransform(content, {
       auxiliaryCommentBefore: ' istanbul ignore next ',
       babelrc: false,
```

Comment:
This call should have a `configFile: false` alongside the `babelrc: false` on it or else it risks applying the user's project-wide config during instrumentation.

Reply:
@loganfsmyth  Perhaps you can help me debug the failing test for this, as it's currently throwing this.

```
  ● ScriptTransformer › writes source map if preprocessor supplies it

    TypeError: Cannot read property 'mtime' of undefined
      at fileMtime (node_modules/@babel/core/lib/config/files/utils.js:39:45)
```

While debugging, I noticed that console logs before the `babelTransform` would display, but not after. I also noticed that `/fruits/package.json` is being `statSync`d, yet that path is NOT defined within the tests. That leads me to believe that it's caused by Babel, or the Istanbul plugin, but after debugging the plugin, it's never even hit. At this point I'm leaning towards it being a Babel config loading problem, but both `babelrc` and `configFile` are false, so I'm a bit lost here.

Reply:
Is something mocking `statSync` maybe? That function should always return an object, or throw: https://github.com/babel/babel/blob/13798feefbf19389cb692aa0b5157276ce7cdf4f/packages/babel-core/src/config/files/utils.js#L21

Reply:
@loganfsmyth We do mock `statSync`, but the weird thing is that `/fruits/package.json` returns undefined while all other calls to `statSync` return the mock. It's very confusing.